corehq/apps/data_dictionary/views.py

```diff
@@ -0,0 +1,74 @@
+from django.core.urlresolvers import reverse
+from django.db.models.query import Prefetch
+from django.http import JsonResponse
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext_lazy as _
+
+from dimagi.utils.decorators.memoized import memoized
+
+from corehq import toggles
+from corehq.apps.domain.decorators import login_and_domain_required
+from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.data_dictionary import util
+from corehq.apps.data_dictionary.models import CaseType, CaseProperty
+from corehq.tabs.tabclasses import ApplicationsTab
+
+
+@login_and_domain_required
+@toggles.DATA_DICTIONARY.required_decorator()
+def generate_data_dictionary(request, domain):
+    util.generate_data_dictionary(domain)
+    return JsonResponse({"status": "success"})
```

Comment:
It provides a quick way to test regenerating the dictionary if you think something is missing. I think this will be taken out in the final form of this